extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -76,10 +80,27 @@ const mode = {
   GEO_API: 3, //       Query API when cache patching unavailable
 };
 
+/**
+ * @enum {string}
+ */
+const countriesWithSubdivision = {
+  US: 'us',
+  CA: 'ca',
+};
+
+/**
+ * Values
+ * @enum {string}
+ */
+const values = {
+  UNKNOWN: 'unknown',
```

Comment:
Nit: no need for an enum here, we can just have a single variable `const UNKNOWN = 'unknown'`

Reply:
just this last change and ill merge @setdream

Reply:
@erwinmombay please check